migrations/staging/20171017104000_stg_court_reporters.js

```diff
@@ -0,0 +1,28 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.withSchema('staging').createTable('court_reporters', function (table) {
+    table.increments('id')
```

Comment:
Should this be in the PR? Need to merge `master` in maybe

Reply:
It's because the merge branch was created before this was branched off master